cypress/integration/lesson_page_spec.js

```diff
@@ -3,8 +3,7 @@ describe('lesson page', () => {
     cy.visit(
       '/lessons/react-using-react-error-boundaries-to-handle-errors-in-react-components',
     )
-    // failing
-    cy.get('#bmpui-id-204').should('be.visible')
+    cy.get('#egghead-player', {timeout: 10000}).should('exist')
```

Comment:
Changed this to look for the player itself. The reason this was failing was that it was timing out before it could find the element. I extended the timeout to be ten seconds and it worked. I'm unsure what would be the best practice here because just extending the timeout feels wrong.

Reply:
I need to look into getting the video player to load faster.

Reply:
It won’t load unless it has the media URLs which load async. We could potentially load them in SSR because we have access to the token. This would make loads of sense, but we would want to make sure that caching doesn’t distribute protected media URLs Willy nilly

Reply:
I also think waiting 10 seconds is totally fine.